Signal/src/views/AvatarImageView.swift

```diff
@@ -1,14 +1,45 @@
-//  Created by Michael Kirk on 12/11/16.
-//  Copyright © 2016 Open Whisper Systems. All rights reserved.
+//
+//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
+//
 
 import UIKit
 
 @IBDesignable
 class AvatarImageView: UIImageView {
```

Comment:
Can we use `AvatarImageView` in the (new group, update group, conversation settings) views?